src/features/AppletSettings/ReportConfigSetting/ReportConfigSetting.tsx

```diff
@@ -0,0 +1,141 @@
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { CheckboxController, EditorController, InputController } from 'components/FormComponents';
+import { StyledBodyLarge, StyledTitleMedium } from 'styles/styledComponents/Typography';
+import theme from 'styles/theme';
+import { Svg } from 'components';
+import { variables } from 'styles/variables';
+
+import { StyledAppletSettingsButton, StyledHeadline } from '../AppletSettings.styles';
+import { defaultValues } from './ReportConfigSetting.const';
+import { reportConfigSchema } from './ReportConfigSetting.schema';
+import {
+  StyledButton,
+  StyledContainer,
+  StyledForm,
+  StyledHeader,
+} from './ReportConfigSetting.styles';
+import { FormValues } from './ReportConfigSetting.types';
+
+export const ReportConfigSetting = () => {
+  const { t } = useTranslation();
+  const isServerConfigured = false;
+  const [isSettingsOpen, setSettingsOpen] = useState(false);
+  const {
+    handleSubmit,
+    control,
+    formState: { isValid },
+  } = useForm({
+    resolver: yupResolver(reportConfigSchema()),
+    defaultValues,
+    mode: 'onChange',
+  });
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  const onSubmit = (values: FormValues) => {};
+
+  return (
+    <StyledForm noValidate onSubmit={handleSubmit(onSubmit)}>
+      <StyledHeader>
+        <StyledHeadline sx={{ marginRight: theme.spacing(2.4) }}>
+          {t('reportConfiguration')}
+        </StyledHeadline>
+        <StyledAppletSettingsButton
+          variant="outlined"
+          type="submit"
+          disabled={!isValid}
+          startIcon={<Svg width="19" height="18" id="save" />}
+        >
+          {t('save')}
+        </StyledAppletSettingsButton>
+      </StyledHeader>
+
+      <StyledContainer>
+        <StyledTitleMedium sx={{ marginBottom: theme.spacing(2.4) }}>
+          {t('emailConfiguration')}
+        </StyledTitleMedium>
+        <InputController
+          name="email"
+          control={control}
+          label={t('respondent')}
+          placeholder={t('respondent')}
```

Comment:
Do we need this and other placeholders?

Reply:
yes, these placeholders there are in design